controllers/inscripciones.go

```diff
@@ -41,8 +41,11 @@ func (c *InscripcionesController) GetEstadoInscripcion() {
 
 	defer errorhandler.HandlePanic(&c.Controller)
 
-	terceroId := c.GetString("persona-id")
-	idPeriodo := c.GetString("id-periodo")
+	// terceroId := c.GetString("persona-id")
+	// idPeriodo := c.GetString("id-periodo")
+
+	terceroId := c.Ctx.Input.Param(":persona_id")
+	idPeriodo := c.Ctx.Input.Param(":id_periodo")
```

Comment:
```suggestion
	terceroId := c.GetString("persona-id")
	idPeriodo := c.GetString("id-periodo")
```
Como lo propones si funciona así ? porque tengo entendido que si en **Param** se pone __query__ debería ir con el `c.GetTipodato("nombre parametro")`

entonce si dice:
```
// @Param	persona-id	query	string	false	"Id del tercero"
terceroId := c.GetString("persona-id")
```

pero si es:
```
// @Param	persona-id	path	string	false	"Id del tercero"
terceroId := c.Ctx.Input.Param(":persona-id")
```